src/features/ITGovernance/Admin/GRBReview/GRBReviewForm/Presentation/index.tsx

```diff
@@ -1,60 +1,517 @@
+                            clearFile={() => field.onChange(null)}
+                          />{' '}
+                        </FormGroup>
+                      );
+                    }}
+                  />
+                </div>
+              </Grid>
+            </Fieldset>
+          </GRBReviewFormStepWrapper>
+        </EasiFormProvider>
+      ) : (
+        <EasiFormProvider<AsyncPresentationFields> {...asyncForm}>
+          <GRBReviewFormStepWrapper
+            grbReview={grbReview}
+            onSubmit={onSubmitAsync}
+            requiredFields={false}
+          >
+            <Fieldset className="body-normal">
+              <Grid desktop={{ col: 6 }}>
+                <div className="border-top-1px border-base-lighter margin-top-3 easi-text-normal">
+                  <h4 className="margin-top-1 margin-bottom-05">
+                    {t('presentationGRBReviewForm.asyncHeading')}
+                  </h4>
+
+                  <p className="margin-y-0 text-base">
+                    {t('presentationGRBReviewForm.asyncDescription')}
+                  </p>
+
+                  <Alert type="info" slim className="margin-top-105">
+                    {t('presentationGRBReviewForm.alert')}
+                  </Alert>
+
+                  <FormGroup
+                    error={
+                      !!errorAsync.asyncRecordingDate
+                        ?.grbReviewAsyncRecordingTime
+                    }
+                    className="margin-top-3"
+                  >
+                    <Controller
+                      name="asyncRecordingDate.grbReviewAsyncRecordingTime"
+                      control={controlAsync}
+                      render={({
+                        field: { ref, ...field },
+                        fieldState: { error }
+                      }) => (
+                        <FormGroup error={!!error}>
+                          <Label htmlFor={field.name} className="text-normal">
+                            {t(
+                              'presentationGRBReviewForm.asyncRecordingDateLabel'
+                            )}
+                          </Label>
+
+                          <HelpText className="margin-top-1">
+                            {t(
+                              'presentationGRBReviewForm.meetingDateDescription'
+                            )}
+                          </HelpText>
+
+                          {!!error?.message && (
+                            <FieldErrorMsg>{t(error.message)}</FieldErrorMsg>
+                          )}
+
+                          <DatePickerFormatted
+                            {...field}
+                            id={field.name}
+                            defaultValue={field.value || ''}
+                            value={field.value || ''}
+                            onChange={e => field.onChange(e || undefined)}
+                            dateInPastWarning
```

Comment:
Are there supposed to be date in past warnings on the date fields? I may have missed that somewhere so just wanted to check.

Reply:
Yes I verified with Zoe earlier we are supposed to have it here